woocommerce/woocommerce-gutenberg-products-block
# Preparation for including Blocks as the default checkout experience in WC 8.3

## What

This change adds a check when replacing shortcodes with classic shortcode block on default cart and checkout pages.

## Why



With Blocks shipping as the default checkout experience on WC 8.3, we need to make sure users on a previous version of WC still get the classic shortcode block, but not after 8.3.
This is done by checking the filter default values for shortcodes.

## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. On a new install, delete the Cart and Checkout pages
2. Go to WooCommerce > Status> Tools  and click  Create Pages 
3. Ensure the created Cart and Checkout pages contain the classic shortcode block (which you can click to transform into blocks)
4. Perform a test order and verify everything works and is using the Cart and Checkout shortcode versions.


## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> n/a
